modules/web/src/app/settings/admin/defaults/template.html

```diff
@@ -212,13 +210,39 @@
           <km-spinner-with-confirmation [isSaved]="isEqual(settings.enableOIDCKubeconfig, apiSettings.enableOIDCKubeconfig)"></km-spinner-with-confirmation>
         </div>
 
+        <div fxLayout="row">
+          <div fxFlex="16%"
+               fxLayoutAlign=" center"
+               class="entry-label">
+            <span>Disable Admin Kubeconfig</span>
+            <div class="km-icon-info km-pointer"
+                 matTooltip="Disable the option to download admin kubeconfig and API endpoints."></div>
```

Comment:
nit: I'm a bit confused about wording here. How about?
```suggestion
                 matTooltip="Disable the option to download admin kubeconfig and corresponding API endpoint."></div>
```

Reply:
Looks better 🚀